packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -105,8 +105,8 @@ export const PaymentConfirmation = ({
             </>
           ) : (
             <>
-              <img src={checkmarkIcon} alt="checkmark icon" />
-              <img src={emailSentIcon} alt="email sent icon" />
+              <img src={checkmarkIcon} alt="" />
+              <img src={emailSentIcon} alt="" />
```

Comment:
👍 
unrelated, but do these need widths/heights to avoid possible reflow and layout shift?